### PR TITLE
Add benchmarks

### PR DESCRIPTION
**Pull request by lino-levan**
Start of resolving #11. Ideally these wouldn't be benchmarks that we made up but this is a good start to proving that skia_canvas is as fast, if not faster (within a few milliseconds) of node alternatives.

### CODE REVIEW

File: bench/deno.js [hunk 1]
```diff
@@ -1,17 +1,52 @@
...
+  const canvas = createCanvasWasm(1024, 512);
+  const ctx = canvas.getContext("2d");
+  drawGradient(ctx);
+  canvas.toBuffer("image/png");
+});
+
+Deno.bench("images: deno skia_canvas", async () => {
+  const canvas = createCanvas(1000, 1400);
+  const ctx = canvas.getContext("2d");
+  const skiaImage = await Image.load("./testdata/skia_logo.png");
```
Comment (DjDeveloperr) [inline bench/deno.js:39]: use `new Image(path)` here, it'll skip deno layer entirely and load image natively

Comment (lino-levan) [reply bench/deno.js:39]: Sounds good, let me update the benchmark.

Comment (lino-levan) [reply bench/deno.js]: Did not look like it had a huge effect, the results are ~5 ms of before.

**Comment by lino-levan** [comment]
@DjDeveloperr any chance that this could get merged?

**Review by DjDeveloperr — APPROVED** [review]
LGTM, thanks!

**Pull request merged**